pkg/storaged/filesystem/utils.jsx

```diff
@@ -85,6 +129,13 @@
         return cockpit.format(_("Mount point is already used for $0"),
                               other_blocks.map(nice_block_name).join(", "));
 
+    if (subvol) {
+        const other_subvols = find_subvols_for_mount_point(client, val, subvol);
+        if (other_subvols.length > 0)
+            return cockpit.format(_("Mount point is already used for subvolume $0"),
+                                  other_subvols.join(", "));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231221-103901-225b81f5-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)